docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     depends_on:
       - "${DB_HOST:-postgresql}"
       - "elasticsearch"
-      - redis-sentinel
```

Comment:
If you do this, is sentinel going to start when we `make run`?

Reply:
I removed the dependency because it is only used in Test environment.
We don't want it when we `make run`.